Description: Use the Ubuntu netboot ramdisk and kernel from installer.
Author: Chuck Short <zulcss@ubuntu.com>
Forwarded: no
--- a/cobbler/modules/manage_import_debian_ubuntu.py
+++ b/cobbler/modules/manage_import_debian_ubuntu.py
@@ -361,9 +361,9 @@
                 self.logger.info("following symlink: %s" % fullname)
                 os.path.walk(fullname, self.distro_adder, distros_added)
 
-            if ( x.startswith("initrd") or x.startswith("ramdisk.image.gz") or x.startswith("vmkboot.gz") ) and x != "initrd.size":
+            if ( x.startswith("initrd.gz") ) and x != "initrd.size":
                 initrd = os.path.join(dirname,x)
-            if ( x.startswith("vmlinu") or x.startswith("kernel.img") or x.startswith("linux") or x.startswith("mboot.c32") ) and x.find("initrd") == -1:
+            if ( x.startswith("linux") ) and x.find("initrd") == -1:
                 kernel = os.path.join(dirname,x)
 
             # if we've collected a matching kernel and initrd pair, turn the in and add them to the list
